packages/grafana-ui/src/utils/typeahead.ts

```diff
@@ -41,15 +41,15 @@ export const calculateListSizes = (theme: GrafanaTheme, allItems: CompletionItem
   };
 };
 
-export const calculateItemHeight = (longestLabelHeight: number, theme: GrafanaTheme) => {
-  const horizontalPadding = parseInt(theme.spacing.sm, 10) * 2;
+export const calculateItemHeight = (longestLabelHeight: number, theme: GrafanaTheme2) => {
+  const horizontalPadding = parseInt(theme.spacing(1), 10) * 2;
```

Comment:
this `parseInt` feels a little funky (i know it was here already 😅), wonder whether we should just change it to be `theme.gridSize` then we don't need to `parseInt`